ml-agents/mlagents/trainers/model_saver/torch_model_saver.py

```diff
@@ -89,7 +89,25 @@ def _load_model(
         policy = cast(TorchPolicy, policy)
 
         for name, mod in modules.items():
-            mod.load_state_dict(saved_state_dict[name])
+            try:
+                mod.load_state_dict(saved_state_dict[name])
+            except Exception:
```

Comment:
What's the specific exception here that we can put? Rather than the generic `Exception`